docs/take-streaming-backup.md

```diff
@@ -59,13 +59,13 @@ were both compressed and encrypted, they must be decrypted before they are uncom
 |---------|------|
 | Stream the backup into an archived named `backup.xbstream` | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`|
 | Stream the backup into a compressed archive named `backup.xbstream`| `$ xtrabackup --backup --stream=xbstream --compress --target-dir=./ > backup.xbstream` |
-| Encrypt the backup | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream gzip -`` | openssl des3 -salt -k “password” backup.xbstream.gz.des3` |
+| Encrypt the backup | `$ xtrabackup --backup --stream=xbstream > backup.xbstream gzip -`` | openssl des3 -salt -k “password” backup.xbstream.gz.des3` |
```

Comment:
I think this command line no longer works. Do you plan to fix this as part of separate PR?

The command that worked for me (using my system config):
`/xtrabackup --user=root --socket=/home/satya/WORK/PS-8.0-trunk/bld-rel/mysql-test/var/tmp/mysqld.1.sock  --backup --stream=xbstream  |gzip  | openssl des3 -salt -k 'password' -out  backup.xbstream.gz.des3`